test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1202,7 +1214,12 @@ def revert_pbsconf(self):
             "PBS_LOG_HIGHRES_TIMESTAMP": None
         }
 
-        self._revert_pbsconf_server(vals_to_set)
+        server_vals_to_set = vals_to_set
+
+        if primary_server.platform == 'shasta':
+            server_vals_to_set["PBS_PUBLIC_HOST_NAME"] = None
```

Comment:
Why are you setting this to None?  Don't you need it set?

Reply:
vals_to_set is weird, we use the keys of the dictionary to determine if the conf var needs to be set. It could be replaced with a list, but that's out of scope.